dbhydra/dbhydra_core.py

```diff
@@ -503,9 +503,7 @@ def create_new_db(self):
         self.connection.commit()
 
     def execute(self, query):
-
-        self.cursor.execute(query)
-        self.connection.commit()
+        return self.cursor.execute(query)
```

Comment:
breaking the way how dbhydra works on many places -> I will modify it with adding parametr "autocommit=True"